### DISCUSSION TITLE
39E7E8E2-070D-4B8D-9FCD-1FBBA54B824B

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#186] [General]
# 39E7E8E2-070D-4B8D-9FCD-1FBBA54B824B

WASP-47, also known as K2-23 and TOI-4539, is a Sun-like star about 870 light-years away in the constellation Aquarius. It has a total number of four planets discovered so far (one superterran, one neptunian, and two jovians)

https://celestia.mobi/addon?item=39E7E8E2-070D-4B8D-9FCD-1FBBA54B824B

**Comment by EfremAlvares** [comment]
Well a new one!
